frontend/src/app/pages/browse/browse.component.less

```diff
@@ -39,38 +39,40 @@ div.browse-container {
     
             div.work-header {
                 margin-bottom: 0;
+                   
                 img.rating {
-                    position: relative;
+                    float: left;
                     width: 24px;
                     height: 24px;
                     border: none;
                     box-shadow: none;
                     top: 0.25rem;
                 }
-                h1 {
-                    display: inline-block;
+                h1 {     
+                    display: inline;
```

Comment:
Making the title and byline `inline` rather than `inline-block` makes it so that the rating image will never occupy a line all by its lonesome.